index.js

```diff
@@ -82,7 +83,7 @@ module.exports = function(source) {
                         cell : bemCell,
                         exist,
                         // prepare path for require cause relative returns us string that we couldn't require
-                        path : requiredPath(path.relative(path.dirname(this.resourcePath), entityPath))
+                        path : winPath(requiredPath(path.relative(path.dirname(this.resourcePath), entityPath)))
```

Comment:
Feels like It's a `requiredPath` trouble, I told you path for require should be with slash (not a backslash) even on windows.

Reply:
no, it's not a problem with `requiredPath`, I checked
Take a look at https://github.com/bem/webpack-bem-loader/issues/51
there are no slashes here `./common.blocksAppRouterAppRouter.js`
it should be `./common.blocks\AppRouter\AppRouter.js` there is nothing wrong with `requiredPath` . It's problem elsewhere

Reply:
Moreover, these guys don't use required-path but unify path too. why?
https://github.com/ant-design/babel-plugin-import/blob/master/src/Plugin.js#L14

Reply:
@zxqfox 
I described all solutions in detals at https://github.com/bem/webpack-bem-loader/issues/26#issuecomment-322731066

So seems @Yeti-or decided to go with `make replace on output of required-path call`

Reply:
> Moreover, these guys don't use required-path but unify path too. why?

Because they are decided to stick to `/` instead of escaping `\` characters.
The problem is path writing to file where it will be evaluated from `'./App\App.js'` to `'./AppApp.js'` as @Guria described in their comment.

These guys also has a special option customName that can return undetermined result (we can't sure the result won't have `\` character) so they just replaces all `\` with `/` that they don't need to escape.

So it's viable solution. But if we'll get some troubles with that in future we should keep that problem in mind.
I'd say we should left a comment in code to this thread at least.